### PR TITLE
Session 2

### PR DESCRIPTION
**Pull request by harishA3**

### CODE REVIEW

File: cricviz/Gemfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
+source 'https://rubygems.org'
```
Comment (abhishekkumar2718) [inline cricviz/Gemfile:1]: You don't need to create your own rails project - edit the existing `session_2/cricviz` folder.

File: session_1/hello_world/Gemfile [hunk 2]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
```
Comment (abhishekkumar2718) [inline session_1/hello_world/Gemfile:1]: Please don't upload session 1 again

File: student_registry/Gemfile [hunk 3]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
```
Comment (abhishekkumar2718) [inline student_registry/Gemfile:1]: You don't need to create your own rails project - edit the existing `session_2/student_registry` folder.

**Review by abhishekkumar2718 — CHANGES_REQUESTED** [review]

File: student_registry/Gemfile (hunk 3 above)
Comment (harishA3) [reply student_registry/Gemfile:1]: Sure

File: session_1/hello_world/Gemfile (hunk 2 above)
Comment (harishA3) [reply session_1/hello_world/Gemfile:1]: I am sorry. Will not do it again

File: cricviz/Gemfile (hunk 1 above)
Comment (harishA3) [reply cricviz/Gemfile:1]: I don't seem to have an existing file session_2/cricviz on the clone repo IRIS-RoR-Bootcamp-2020

Comment (abhishekkumar2718) [reply cricviz/Gemfile:1]: Sync your fork: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

File: session_1/hello_world/Gemfile (hunk 2 above)
Comment (abhishekkumar2718) [reply session_1/hello_world/Gemfile:1]: Can you remove it from this MR too?

File: cricviz/Gemfile (hunk 1 above)
Comment (harishA3) [reply cricviz/Gemfile:1]: thank you

File: session_1/hello_world/Gemfile (hunk 2 above)
Comment (harishA3) [reply session_1/hello_world/Gemfile:1]: could you please explain what you mean by MR?

**Pull request closed without merging**